Makefile

```diff
@@ -179,9 +183,11 @@ endif
 endif
 
 # you need latest valgrind, otherwise you might not get line numbers in your valgrind output
+valgrind-run: TARGET=$(BIN_OUTPUT_PATH)/$(PROGRAM)
```

Comment:
Made this generic so we can run any c program with args against valgrind.

Reply:
Nice!